social/src/main/java/com/amity/socialcloud/uikit/feed/settings/AmitySocialUISettings.kt

```diff
@@ -20,6 +21,12 @@ object AmitySocialUISettings {
     private var postViewHolders: MutableMap<String, AmityPostRenderer> =
         AmityDefaultPostViewHolders.getDefaultMap()
 
+    lateinit var postMarkupProcessor: PostMarkupProcessor
```

Comment:
Looking at the API of this settings object, it seems risky to have a `lateinit` here and it being used directly as a global singleton - at some point it can happen that this variable is not initialised when used.
It would probably be better to provide the processor in a constructor or through DI where used, but if this is an overkill, I would vote having it as nullable or with a default a no-op processor (which could also log a Firebase warning/error if used, without crashing the app).